documentation/operators/src/introduction.md

```diff
@@ -0,0 +1,34 @@
+# Introduction
+
+This is Nym's operators guide, containing information and setup tutorials about the various pieces of Nym mixnet infrastructure nodes, such as mix node, gateway and network requester.
+
+If you are new to Nym and want to learn about the mixnet, how to integrate with the network, developer tutorials and quickstart guides, check out the [Developer Portal](https://nymtech.net/developers/).
+
+If you want to go deeper into Nym architecture and various software such as different, application clients, existing applications like the desktop wallet, mixnet explorer or SDKs, visit [Nym's technical documentation](https://nymtech.net/docs/).
+
+If you are looking for information on grants and beta-release Nym apps, check out the [Shipyard](https://shipyard.nymtech.net) site.
+
+
+## Popular pages
+
+<!---serinko: not sure how much to include from the docs and developers books in here--->
```

Comment:
I think for the moment we do: 

* node setup guides 
* node maintenance and troubleshooting 

I'm not sure we need client setup... maybe keep that in the docs